cwl/src/main/scala/cwl/model.scala

```diff
@@ -273,6 +275,16 @@ case class ResourceRequirement(
   def effectiveOutdirMax = outdirMax.orElse(outdirMin)
 }
 
+/**
+  * This promotes InputResourceRequirement to a first class citizen requirement, which it really isn't.
+  * Since it's the only one for now it's not a big deal but if more of these pop up we might want to treat custom requirements
+  * in a different way
+  */
```

Comment:
Hmm so this is really a hint that Cromwell now supports?

Reply:
Exactly

Reply:
Yes, the reason behind this comment is that I saw this other comment: https://github.com/broadinstitute/cromwell/blob/913efc91927083788dec8f87e70c1ab22d800b00/cwl/src/main/scala/cwl/Tool.scala#L81
Which led me to think that hints/requirements that are not defined in the CWL Spec were meant to be handled differently than treating them as a "real" `Requirement`

Reply:
Oh that may be an unclear comment. IIRC hints can be of any type, I think the comment is trying to say that if they're of a type other than `Requirement` they're going to be dropped on the floor. But `Requirement` hints should be handled just fine.